src/pages/ProfilePage/ProfileDetails.tsx

```
@@ -138,6 +220,541 @@ export default function ProfileDetails() {
+                    className="h-12 w-12 rounded-md border text-center text-lg"
+                  />
+                </div>
+                <button
+                  onClick={handleVerifyCode}
+                  className="mb-4 w-full rounded-md bg-gray-200 py-2 text-gray-500"
+                >
+                  Verify
+                </button>
+                <p className="text-sm text-gray-600">
+                  Didn't receive the code?{" "}
+                  <a href="#" className="text-blue-600">
```

Biome (2.3.13)

[error] Provide a valid value for the attribute href.

The href attribute should be a valid a URL
Check this thorough explanation to better understand the context.

(lint/a11y/useValidAnchor)